lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -140,8 +140,17 @@ sub group_overview {
     return $self->reply->not_found unless $group;
 
     my $res = $self->_group_result($group, $limit_builds);
-    my @comments = $group->comments->all;
-    for my $comment (@comments) {
+    my @comments;
+    my @pinned_comments;
+    for my $comment ($group->comments->all) {
+        # find pinned comments
+        if ($comment->user->is_operator && CORE::index($comment->text, 'pinned-description') >= 0) {
```

Comment:
I'm still missing something.
Logged in users can add comments. Any logged in user thus can create comment with 'pinned-comment' keyword, right? Then if you are regular users you don't see them as pinned, but when you are operator you see them pinned? This doesn't make sense to me.